Ranga: políticas do sistema em /etc/ranga/policies (as do Chromium não valem pro Ranga)

--- a/components/policy/core/common/policy_paths.cc
+++ b/components/policy/core/common/policy_paths.cc
@@ -18,7 +18,7 @@
 #elif BUILDFLAG(GOOGLE_CHROME_FOR_TESTING_BRANDING)
 const char kPolicyPath[] = "/etc/opt/chrome_for_testing/policies";
 #else
-const char kPolicyPath[] = "/etc/chromium/policies";
+const char kPolicyPath[] = "/etc/ranga/policies";
 #endif  // BUILDFLAG(GOOGLE_CHROME_BRANDING)
 #endif  // BUILDFLAG(IS_POSIX) && !BUILDFLAG(IS_MAC)
 
